bigchaindb/commands/bigchaindb.py

```diff
@@ -317,41 +315,38 @@ def create_parser():
                                help='The backend to use. It can only be '
                                '"localmongodb", currently.')
 
-    # parser for managing validator elections
-    validator_parser = subparsers.add_parser('upsert-validator',
-                                             help='Add/update/delete a validator.')
-
-    validator_subparser = validator_parser.add_subparsers(title='Action',
-                                                          dest='action')
-
-    new_election_parser = validator_subparser.add_parser('new',
-                                                         help='Calls a new election.')
+    # parser for managing elections
+    election_parser = subparsers.add_parser('election',
+                                            help='Manage elections.')
 
-    new_election_parser.add_argument('public_key',
-                                     help='Public key of the validator to be added/updated/removed.')
+    election_subparser = election_parser.add_subparsers(title='Action',
+                                                        dest='action')
 
-    new_election_parser.add_argument('power',
-                                     type=int,
-                                     help='The proposed power for the validator. '
-                                          'Setting to 0 will remove the validator.')
+    new_election_parser = election_subparser.add_parser('new',
```

Comment:
@kansi has [suggested](https://github.com/bigchaindb/BEPs/pull/82#discussion_r216947470) to be more concise and just go with `bigchaindb election <type> ..` (no `new`). I like the idea.

Reply:
Discussed offline - removing it does not seem to be worth the effort so let's keep it.